### PR TITLE
allow user to view connector instance configuration

### PR DESCRIPTION
**Pull request by konstan**
Connected to #1401

### CODE REVIEW

File: cimi-resources/src/com/sixsq/slipstream/ssclj/resources/service_attribute_namespace.clj [hunk 1]
```diff
@@ -89,7 +89,7 @@
 (defmethod crud/add resource-name
   [{{:keys [prefix uri]} :body :as request}]
   (if-let [id (colliding-id prefix uri)]
-    (response/response-conflict id)
+    (response/ex-conflict id)
```
Comment (loomis) [inline cimi-resources/src/com/sixsq/slipstream/ssclj/resources/service_attribute_namespace.clj]: What's the reason for this change?  For me, this causes the lifecycle tests for the Service Attribute Namespace to fail.

Comment (konstan) [reply cimi-resources/src/com/sixsq/slipstream/ssclj/resources/service_attribute_namespace.clj]: It was failing for me with `response/response-conflict`, hence the change.  I might need to recompile the `response` dependency and retry.  I'll update you shortly.

Comment (konstan) [reply cimi-resources/src/com/sixsq/slipstream/ssclj/resources/service_attribute_namespace.clj]: screen shot 2018-02-20 at 08 22 44

Comment (konstan) [reply cimi-resources/src/com/sixsq/slipstream/ssclj/resources/service_attribute_namespace.clj]: Changed back to `response/response-conflict`.  All compiles and tests pass on my side.
